api/services/search-service.js

```diff
@@ -2,39 +2,39 @@
 
 var Client = require('node-rest-client').Client;
 var client = new Client();
-// const GRIDDY_BASE = "http://griddy.org/api";
-const GRIDDY_BASE = "http://localhost:3001/griddy"; // MOCK URL
+
+var today = new Date();
+var year = today.getFullYear();
+const MONTH = "9";
+const UOFT_BASE = "https://timetable.iit.artsci.utoronto.ca/api/" + year + MONTH + "/courses?";
```

Comment:
will we have to keep manually updating the month? If so, is there a better implementation?